WordPress/AbstractValidSlugSniff.php

```diff
@@ -0,0 +1,276 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MITnamespace WordPressCS\WordPress;
+ */
+
+namespace WordPressCS\WordPress;
+
+use PHP_CodeSniffer\Util\Tokens;
+use PHPCSUtils\Tokens\Collections;
+use PHPCSUtils\Utils\PassedParameters;
+use PHPCSUtils\Utils\TextStrings;
+use WordPressCS\WordPress\AbstractFunctionParameterSniff;
+
+/**
+ * Validates names passed to a function call.
+ *
+ * Checks slugs for the presence of invalid characters, excessive length,
+ * and reserved keywords.
+ */
```

Comment:
What is the correct version number for new `@since` tags? Or is it possible to use placeholders that are automatically replaced when changes are released?

Reply:
For a new sniff, the next minor (unless a major is planned to be next, in which case, the next major).
There is no automatic replacement, so if the PR is not ready for merge in time for the next minor, it means the `@since` tags may need to be updated by the time the PR _is_ ready for merge.

Reply:
Ok, thank you